electrode-ota-server-dao-mariadb/src/client.js

```diff
@@ -0,0 +1,87 @@
+import _ from 'lodash';
+import {Sequelize} from 'sequelize';
+import {ProxyModelWrapper} from './proxy';
+import cls from 'continuation-local-storage';
```

Comment:
This. Will. Be. Your. Worst. Nightmare.

https://github.com/othiym23/node-continuation-local-storage/issues/59